plugins/accounts/plugins/accounts/less/hawkular-accounts.less

```diff
@@ -11,3 +11,7 @@
   color: #fff;
   font-weight: bold;
 }
+
+body {
+  padding-top: 0;
+}
```

Comment:
On this case, I think it's "safe", as Patternfly resets it, so, we are just restoring what toastr did. But it would be nice to get someone ( @cardosogabriel  ? @ammendonca ?) to check it.